exercises/reverse-string/README.md

```diff
@@ -0,0 +1,40 @@
+# Reverse String
+
+Reverse a string
+
+For example:
+input: "cool"
+output: "looc"
+
+## Setup
+
+Go through the setup instructions for JavaScript to install the
```

Comment:
You'll want to update these references so the point to the EcmaScript track for now.  We'll need to handle converting them (finally) to JavaScript in the migration.